BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/Model/Submission.java

```diff
@@ -0,0 +1,172 @@
+package com.blueprint.foe.beetracker.Model;
+
+import android.location.Location;
+
+import java.util.MissingFormatArgumentException;
+
+/**
+ * Represents the data object that contains all information related to submission.
+ */
+
+public class Submission {
+    public static final int MAX_FACE = 3;
+    public static final int MAX_THORAX = 3;
+    public static final int MAX_ABDOMEN = 3;
+
+    private int face; // the index of the image chosen by user
+    private int thorax; // the index of the image chosen by user
+    private int abdomen; // the index of the image chosen by user
```

Comment:
If we add a new face, we just have to change MAX_FACES. There are no names associated with these values they just correspond to image indices. The enums would probably be like FACE { ZERO, ONE, TWO } which I don't think is worth encapsulating.